### PR TITLE
Integration tests

### LINKED ISSUE #14: Functional Tests
# Need
* As a developer on the Query Service
* I want to be confident my changes wont break current functionality

# Approach
- [ ] ???????

### LINKED ISSUE #92: Refactor functional tests into integration test runner
As described in the README file.

### PR DESCRIPTION
**Pull request by kislyuk**
Closes #92 
Closes #14

### CODE REVIEW

File: tests/integration/test_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    def setUp(self):
+        self.api_url = f"https://{os.environ['API_DOMAIN_NAME']}"
+        self.app = requests
+        logger.warning(f"\n\nTESTING ENVIRONMENT {config.stage} at URL {self.api_url}. \n")
+
+    def test_health_check(self):
+        logger.warning("CHECK SERVICE HEALTH")
+        self.assertGetResponse(f"{self.api_url}/internal/health", expected_code=requests.codes.ok)
+
+    def test_query(self):
```
Comment (MDunitz) [inline tests/integration/test_endpoints.py]: can we add a test for queries submitted to /query that result in the creation of a job?

File: tests/unit/test_create_async_query.py [hunk 2]
```diff
@@ -31,3 +31,7 @@ def test_process_async_query_keeps_job_status_updated(self, mock_job_status):
     def test_create_async_query_calls_process_async_func_with_correct_args(self, mock_process):
         # handler(self.mock_event, 'context')
```
Comment (MDunitz) [inline tests/unit/test_create_async_query.py]: delete?

File: tests/integration/test_endpoints.py (hunk 1 above)
Comment (kislyuk) [reply tests/integration/test_endpoints.py]: Yes, definitely. But I would prefer to add that in a separate PR.

File: tests/unit/test_create_async_query.py (hunk 2 above)
Comment (kislyuk) [reply tests/unit/test_create_async_query.py]: The change in this file was erroneusly included in this PR. I updated it to exclude this change. I will update the tests in this file in a separate PR.

Comment (kislyuk) [reply tests/unit/test_create_async_query.py]: Filed the PR in #161.

**Pull request merged**